src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -138,41 +174,63 @@ export class SchoolSelectField extends React.Component {
-                onKeyDown={this.handleSearchInputKeyDown}
-                ref={input => { this.searchInput = input; }}
-                type="text"
-                value={searchInputValue}/>
-              <button
-                className="search-schools-button usa-button-primary"
-                onClick={this.handleSearchClick}>
-                {'Search Schools'}
-              </button>
-            </div>
-            <div className="clear-search">
-              <button
-                className="va-button-link start-over"
-                onClick={this.handleStartOver}>
-                {'Start Over'}
-              </button>
+          <div className={schoolSearchClassNames}>
+            <label
+              id="school-search-label"
+              className={labelClassNames}
+              htmlFor="school-search-input">
+              {'School Information'}
+              {!manualSchoolEntryChecked && <span className="schemaform-required-span">{'(*Required)'}</span>}
+            </label>
+            {showErrors && <span
+              className="usa-input-error-message"
+              role="alert"
+              id="facility-code-error-message">
+              {errorMessages.map((message, index) => (
+                <span key={index}><span className="sr-only">Error</span>{message}</span>))
+              }
+            </span>}
+            <span>
+              {'Enter your school’s name or city to search for your school'}
+            </span>
+            <div className="search-controls">
+              <Element name="schoolSearch"/>
+              <div className="search-input">
+                <input
+                  name="school-search-input"
+                  onChange={this.handleSearchInputChange}
+                  onKeyDown={this.handleSearchInputKeyDown}
+                  ref={input => { this.searchInput = input; }}
+                  type="text"
+                  value={searchInputValue}/>
+                <button
+                  className="search-schools-button usa-button-primary"
+                  onClick={this.handleSearchClick}>
+                  {'Search Schools'}
+                </button>
+              </div>
+              <div className="clear-search">
+                <button
+                  className="va-button-link start-over"
+                  onClick={this.handleStartOver}>
+                  {'Start Over'}
+                </button>
+              </div>
             </div>
           </div>
+          <ErrorableCheckbox
+            checked={manualSchoolEntryChecked}
+            onValueChange={() => this.handleManualSchoolEntryToggled(manualSchoolEntryChecked)}
+            label={<span>I’d rather type in my school information</span>}/>
           <div
             aria-live="polite"
             aria-relevant="additions text">
-            {searchResultsCount > 0 && <span>
+            {showSearchResults && searchResultsCount > 0 && <span>
```

Comment:
Went with using a single prop in several places to handle the hiding of the search results when the manual entry is visible. I wanted to retain the state behind the scenes to keep it friendly- you can check and uncheck the box without losing any of your search state.